hedera-node/src/main/java/com/hedera/services/throttling/ExpiryThrottle.java

```diff
@@ -47,34 +47,37 @@ public class ExpiryThrottle {
 
     private final HybridResouceLoader resourceLoader;
 
-    private DeterministicThrottle throttle;
-    private Map<MapAccessType, Integer> accessReqs;
+    private long usageCapForCycleEntry;
+    private ThrottleConfig config;
 
     @Inject
     public ExpiryThrottle(final HybridResouceLoader resourceLoader) {
         this.resourceLoader = resourceLoader;
     }
 
-    public boolean allow(final List<MapAccessType> accessTypes) {
-        return allow(accessTypes, null);
+    public boolean stillLacksMinFreeCapAfterLeakingUntil(final Instant now) {
+        if (config == null) {
+            return true;
+        }
+        config.throttle.leakUntil(now);
+        return config.throttle.used() > usageCapForCycleEntry;
```

Comment:
nit : easier to understand if it is `confg.throttle.capacityFree()` < `minFreeReq` from `config.throttle.clampedCapacityRequiredFor(requiredOps(minUnitOfWork))`

Reply:
Agreed, fixed.